component/tests/vshn.yml

```diff
@@ -55,9 +55,16 @@ parameters:
         emailAlerting:
           enabled: true
           smtpPassword: "whatever"
+        services:
+          mariadb:
+            enabled: true
+            grpcEndpoint: 172.18.0.1:9443
```

Comment:
Don't forget to comment out in the end. Although the productive deployment won't actually look at this.

Reply:
This has already been refactored accordingly